### DISCUSSION TITLE
Facing : 401 The current user has insufficient permissions to perform the requested o (truncated...)

### DISCUSSION THREAD

**Discussion by harshchhantbar** [imdhemy/laravel-in-app-purchases#113] [Q&A]
Please, Help me to solve this error.
I have created service account with owner role and set json file path on env but facing this error
"error": {
    "code": 401,
    "message": "The current user has insufficient permissions to perform the requested o (truncated...)
 in file /home/projects/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php on line 113

**Comment by geekymindio** [comment]
same problem, any help?

**Comment by imdhemy** [comment]
duplication of #94
